benches/function.rs

```diff
@@ -1,6 +1,10 @@
 use std::os::raw::c_void;
 
 fn main() {
+  // Skip running benchmarks in debug or CI.
```

Comment:
Please add some more detail here. "For an unknown reason running these benchmarks in CI on aarch64 linux hangs. Skipping them for now."

Reply:
Oops. Missed this because of "auto-merge"